frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelServingGlobal.cy.ts

```diff
@@ -394,12 +458,42 @@ describe('Model Serving Global', () => {
     inferenceServiceModal.findLocationBucketInput().type('test-bucket');
     inferenceServiceModal.findLocationPathInput().type('test-model/');
     inferenceServiceModal.findSubmitButton().should('be.enabled');
+
+    inferenceServiceModal.findSubmitButton().click();
+
+    cy.wait('@createInferenceService').then((interception) => {
+      expect(interception.request.url).to.include('?dryRun=All'); //dry run request
+      expect(interception.request.method).to.eql('POST');
+      expect(interception.request.body).to.eql({
```

Comment:
Can you use mockInferenceServiceK8sResource({
          name: 'test-name',
          path: 'test-model/',
          displayName: 'Test Name',
          isModelMesh: true,
        }) function?

Reply:
Here I am asserting the request it does not have all the properties of mock response. So thought of not using those.